[rospy.client][INFO] 2021-11-25 14:57:57,144: init_node, name[/server_endpoint], pid[14573]
[xmlrpc][INFO] 2021-11-25 14:57:57,144: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-11-25 14:57:57,144: Started XML-RPC server [http://wng:38703/]
[rospy.init][INFO] 2021-11-25 14:57:57,145: ROS Slave URI: [http://wng:38703/]
[rospy.impl.masterslave][INFO] 2021-11-25 14:57:57,145: _ready: http://wng:38703/
[xmlrpc][INFO] 2021-11-25 14:57:57,145: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2021-11-25 14:57:57,145: Registering with master node http://localhost:11311
[rospy.init][INFO] 2021-11-25 14:57:57,245: registered with master
[rospy.rosout][INFO] 2021-11-25 14:57:57,245: initializing /rosout core topic
[rospy.rosout][INFO] 2021-11-25 14:57:57,246: connected to core topic /rosout
[rospy.simtime][INFO] 2021-11-25 14:57:57,247: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-11-25 14:57:57,253: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-11-25 14:57:57,515: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-11-25 14:58:18,708: Connection from 192.168.1.146
[rosout][INFO] 2021-11-25 14:58:18,712: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-11-25 14:58:18,772: topic[/execution_check] adding connection to [http://wng:38703/], count 0
[rospy.internal][INFO] 2021-11-25 14:58:18,773: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-11-25 16:01:59,869: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-11-25 16:01:59,873: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-11-25 16:01:59,873: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-11-25 16:01:59,874: topic[/execution_check] removing connection to http://wng:38703/
[rospy.impl.masterslave][INFO] 2021-11-25 16:01:59,874: signal-2
[rospy.internal][ERROR] 2021-11-25 16:01:59,874: Unable to initiate TCP/IP socket to wng:33699 (http://wng:38703/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.core][INFO] 2021-11-25 16:01:59,983: signal_shutdown [atexit]
